content/docs/06-integrations/00-aws-autoscaler.md

```diff
@@ -14,81 +14,208 @@ import InfoBox from 'shared/components/InfoBox';
 import Tooltip from "shared/components/ui/Tooltip";
 
-# AWS Cluster Autoscaler
-
-Palette supports Autoscaling for our **EKS clusters** to maximise the benefits of the AWS Cloud utilization. Palette Auto Scaling ensures that instances are launched propotional to the workload on the EKS cluster. The resources are scaled up and down based on the changing workload on the cluster by continously monitoring the applications and automatically adjusts the capacity to maintain steady, predictable performance at the lowest possible cost.
-
-
-# Prerequisite
-
-* Kubernetes version 1.19.x and above.
-* Full Cluster Autoscaler Policy for the service account.
-* Update the Kubernetes Pack node group `managedMachinePool` with the **ARN** of the autoscaler policy created for the service account.
-
-
-## Full Cluster Autoscaler Features Policy (Recommended)
-
-```json
-{
-  "Version": "2012-10-17",
-  "Statement": [
-    {
-      "Effect": "Allow",
-      "Action": [
-        "autoscaling:DescribeAutoScalingGroups",
-        "autoscaling:DescribeAutoScalingInstances",
-        "autoscaling:DescribeLaunchConfigurations",
-        "autoscaling:DescribeTags",
-        "ec2:DescribeInstanceTypes",
-        "ec2:DescribeLaunchTemplateVersions"
-      ],
-      "Resource": ["*"]
-    },
-    {
-      "Effect": "Allow",
-      "Action": [
-        "autoscaling:SetDesiredCapacity",
-        "autoscaling:TerminateInstanceInAutoScalingGroup",
-        "ec2:DescribeInstanceTypes",
-        "eks:DescribeNodegroup"
-      ],
-      "Resource": ["*"]
-    }
-  ]
-}
-```
-<InfoBox>
-
-To deploy AWS Autoscaler, update the Kubernetes pack `managedMachinePool` node group with the **ARN** of the autoscaler policy created for the service account.
-
-```yaml
-managedMachinePool:
-  #roleName: {{ name of the self-managed role | format "${string}" }}
-
-  ## A list of additional policies to attach to the node group role
-  roleAdditionalPolicies:
-  - "arn:aws:iam::012345678910:policy/autoscalingpolicy"
-```
-</InfoBox>
-
-
-## Versions Supported
+# AWS Cluster Autoscaler 
+
+Palette supports autoscaling for EKS clusters by using the AWS Cluster Autoscaler pack. 
+The Cluster Autoscaler dynamically scales cluster resources. It monitors the workload and provisions or shuts down cluster nodes to maximize cluster utilization. It resizes the Kubernetes cluster under the following conditions:
+<br />
+
+* Insufficient resources leading to multiple pod failures in the cluster. The Cluster Autoscaler redistributes these pods to different nodes in such cases.
+
+
+* Underutilized cluster nodes for a specific period. In this scenario, the Cluster Autoscaler migrates the pods from underutilized nodes to other available nodes.
+
+
+Cluster Autoscaler pack runs as a `Deployment` in your cluster and utilizes [Amazon EC2 Auto Scaling Groups](https://docs.aws.amazon.com/autoscaling/ec2/userguide/what-is-amazon-ec2-auto-scaling.html) to manage node groups. 

```

Comment:
```suggestion
Cluster Autoscaler pack is deployed as a [*Deployment*](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) in your cluster and utilizes [Amazon EC2 Auto Scaling Groups](https://docs.aws.amazon.com/autoscaling/ec2/userguide/what-is-amazon-ec2-auto-scaling.html) to manage node groups. 
```

Run is an ableist word, so we avoid those. The *Deployment* is a reference to the Kubernetes object, Deployment, so we can italize it so that it stands out and not associated with the regular English word "Deployment."